Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2113,7 +2113,27 @@ extension BrowserViewController: ContextMenuHelperDelegate {
             let copyAction = UIAlertAction(title: copyImageTitle, style: UIAlertActionStyle.Default) { (action: UIAlertAction) -> Void in
                 let pasteBoard = UIPasteboard.generalPasteboard()
                 pasteBoard.string = url.absoluteString
-                // TODO: put the actual image on the clipboard
+                // put the actual image on the clipboard
+                // do this be asyncronously just in case we're in a low bandwidth situation
+                let application = UIApplication.sharedApplication()
+                var taskId: UIBackgroundTaskIdentifier = 0
+                taskId = application.beginBackgroundTaskWithExpirationHandler { _ in
+                    application.endBackgroundTask(taskId)
+                }
+
+                dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0)) {
```

Comment:
Nit: Don't need to wrap the Alamofire.request call with dispatch_async since the request will happen on it's own background queue internally.